litellm/litellm_core_utils/exception_mapping_utils.py

```diff
@@ -274,7 +274,7 @@ def exception_type(  # type: ignore  # noqa: PLR0915
                         + "Exception"
                     )
 
-                if "429" in error_str:
+                if "429" in error_str or "rate limit" in error_str.lower():
```

Comment:
working on improving code quality in this file. Can we have a class called ExceptionCheckers and then add a static method that checks is_error_str_rate_limit()

Reply:
also can you confirm with a manual test to fireworks that we end up mapping it correctly ?